frontend/src/metabase/components/Popover/TippyPopover.tsx

```diff
@@ -83,6 +94,7 @@ function TippyPopover({
   const [mounted, setMounted] = useState(!lazy);
   const shouldShowContent = mounted && content != null;
   const isControlled = props.visible != null;
+  const [maxHeight, setMaxHeight] = useState<number>();
```

Comment:
I need a way to pass `maxHeight` to the children otherwise, keyboard navigation wouldn't work. See more at https://github.com/metabase/metabase/pull/22483/files#r866697348